lib/streamlit/state/session_state.py

```diff
@@ -148,7 +145,22 @@ def _get_session_state() -> SessionState:
             " be conflicting with our system."
         )
 
-    return this_session.session_state
+    return this_session
+
+
+def get_session_state() -> SessionState:
```

Comment:
Since this is internal, why not have it start with an underscore?

Reply:
I dropped the underscore since it's now being imported/used in `elements/utils.py`